### PR TITLE
KResNet34 || CIFAR10 || chen

### PR DESCRIPTION
**Pull request by wang-chen**

### CODE REVIEW

File: CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "Chen Wang",
+    "authorEmail": "wang.chen@zoho.com",
+    "framework": "PyTorch",
+    "codeURL": "https://github.com/wang-chen/KervNets",
+    "model": "KervResNet34",
+    "hardware": "1 GPU (Nvidia GeForce GTX 1080 Ti)",
+    "costPerHour": "NA",
```
Comment (codyaustun) [inline CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json]: If you're not submitting to cost, remove this field.

File: CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "Chen Wang",
+    "authorEmail": "wang.chen@zoho.com",
+    "framework": "PyTorch",
```
Comment (codyaustun) [inline CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json]: What version of PyTorch did you use? Please at the version number or git commit hash.

File: CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "codeURL": "https://github.com/wang-chen/KervNets",
+    "model": "KervResNet34",
+    "hardware": "1 GPU (Nvidia GeForce GTX 1080 Ti)",
+    "costPerHour": "NA",
+    "timestamp": "2018-04-20",
+    "misc": {
+        "CUDA version": "8.0"
```
Comment (codyaustun) [inline CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json]: What hyperparameters did you use? Please add the following information to `misc`:

- learning rate schedule
- batch size
- optimizer
- momentum
- weight decay

The [Google ResNet50 submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/ImageNet/train/google_resnet50_tpu_tensorflow18.json) is a good example for reference.

File: CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "Chen Wang",
+    "authorEmail": "wang.chen@zoho.com",
+    "framework": "PyTorch",
+    "codeURL": "https://github.com/wang-chen/KervNets",
```
Comment (codyaustun) [inline CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json]: Please include the commit hash in the URL or add an extra field to `misc`.

**Review by codyaustun — CHANGES_REQUESTED** [review]

**Comment by wang-chen** [comment]
Thanks for your time, I have added the details following your comments.

File: CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "timestamp": "2018-04-20",
+    "misc": {
+        "CUDA version": "8.0",
+        "optimizer": "SGD with Momentum",
+    	"momentum": 0.9,
+    	"weightDecay": 0.0001,
+    	"batchSize": 128,
+    	"epoch": 200,
+    	"LR": 0.1,
+    	"MultiStepLR_Milestones": "[50,100,150]",
```
Comment (codyaustun) [inline CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json]: Please remove the `LR`, `MultiStepLR_Milestones`, and `MultiStepLR_Gamma` fields and express the learning rate schedule using the format below:

```
misc: {
  ...
  schedule: [
    {"learning_rate": 0.1, "epochs": 50},
    {"learning_rate": 0.01, "epochs": 50},
    {"learning_rate": 0.001, "epochs": 50},
    {"learning_rate": 0.0001, "epochs": 50},
  ]
}
```

**Review by codyaustun — CHANGES_REQUESTED** [review]

**Comment by wang-chen** [comment]
Thanks for your time. It has been changed accordingly.

**Review by codyaustun — APPROVED** [review]
